CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Minio cookbook contribution guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+Minio cookbook is the collection of detailed guides to set up Minio with other tools/platforms. Minio team adds new receipies to the cookbook, however, if you have a working setup that uses Minio with other tools, we welcome you to add your receipe. 
+
+Before creating a PR, please check that no similar receipe is already available. Also, please add the below setions in your guide before adding a Cookbook PR. 
```

Comment:
> How about instead of asking them to add the following sections, we can ask the user to follow the format?

Agreed. Updated the sentence.

> recipes instead receipies (twice in the first paragraph and once in second paragraph). section instead of setion

Fixed.